### PR TITLE
this will complete ticket-11-api-articles-topic-query

### PR DESCRIPTION
**Pull request by tomfollows**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -339,3 +339,60 @@ describe("DELETE /api/comments/:comment_id", () => {
...
+  });
+});
+  test("404: will respond with a 404 when the server cannot find the requested resource", () => {
+    return request(app)
+      .get("/api/articles?topic=mit")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Topic Not Found");
+      });
+  });
```
Comment (nc-nick-jones) [inline __tests__/api.test.js:398]: let's have one extra test for a topic that exists but has no articles - 200, empty array.

**Review by nc-nick-jones — COMMENTED** [review]
excellent work! one more test I think we need.
